tools/cluster-deploy/cluster-deploy.go

```diff
@@ -6,36 +6,34 @@ import (
 	"log"
 	"os"
 
-	deploymanager "github.com/openshift/ocs-operator/pkg/deploy-manager"
+	deploymanager "github.com/openshift/ocs-operator/functests"
```

Comment:
What I am not too sure about is whether this is actually a clean approach: to import the functests in the tool and use the variables and the cmdline params defined in the functest... Shouldn't the params be defined in the tool?

Reply:
@jarrpa ^^

Reply:
...yeah, no, that ain't right. Sorry I didn't catch this earlier. The ONLY line that should change in this file is line 9. Everything else should be reverted.